### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'croptopia'!

### DISCUSSION THREAD

**Discussion by n0tcharli3** [FabricMC/fabric-api#2772] [Player support]
---- Minecraft Crash Report ----
// My bad.

Time: 2022-12-20 11:48:00
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'croptopia'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:459)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	Suppressed: java.lang.ExceptionInInitializerError
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:467)
		at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50)
		at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117)
		at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
		at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
		... 8 more
	Caused by: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
		at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:226)
		at com.google.gson.Gson.fromJson(Gson.java:963)
		at com.google.gson.Gson.fromJson(Gson.java:928)
		at com.google.gson.Gson.fromJson(Gson.java:877)
		at com.google.gson.Gson.fromJson(Gson.java:848)
		at space.essem.image2map.config.Image2MapConfig.loadOrCreateConfig(Image2MapConfig.java:30)
		at space.essem.image2map.Image2Map.<clinit>(Image2Map.java:51)
		... 14 more
	Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
		at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:384)
		at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:215)
		... 20 more
Caused by: java.lang.NullPointerException: Cannot invoke "org.spongepowered.configurate.ConfigurationNode.node(Object[])" because the return value of "com.epherical.croptopia.config.CroptopiaConfig.getRootNode()" is null
	at com.epherical.croptopia.generator.BiomeModifiers.init(BiomeModifiers.java:29)
	at com.epherical.croptopia.Croptopia.onInitialize(Croptopia.java:98)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 8 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:459)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22000.832 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22000.1165 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2211.5 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:MicrosoftÂ® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22000.1042 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22000.778 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.22000.258 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22000.1335 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jna15322763442836488940.dll:JNA native library:6.1.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.22000.71 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		msvcp140.dll:MicrosoftÂ® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:MicrosoftÂ® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:Power Profile Helper DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ucrtbase.dll:MicrosoftÂ® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:MicrosoftÂ® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.22000.1335 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.22000.184 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 476077984 bytes (454 MiB) / 1409286144 bytes (1344 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Pentium(R) Silver N5030 CPU @ 1.10GHz
	Identifier: Intel64 Family 6 Model 122 Stepping 8
	Microarchitecture: unknown
	Frequency (GHz): 1.09
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: Intel(R) UHD Graphics 605
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x3184
	Graphics card #0 versionInfo: DriverVersion=30.0.101.2079
	Memory slot #0 capacity (MB): 4096.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 10821.17
	Virtual memory used (MB): 8845.93
	Swap memory total (MB): 6909.84
	Swap memory used (MB): 1832.46
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		architectury: Architectury 6.3.56
		bclib: BCLib 2.1.2
		betteranimalsplus: Better Animals Plus 1.19-11.0.7
		betterend: Better End 2.1.1
		bitsandchisels: BitsAndChisels 2.7.0
			stacc: Stacc 1.3.4
		croptopia: Croptopia 2.1.2
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		extraorigins: Extra Origins 1.19-2
		fabric-api: Fabric API 0.66.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.12+93d8cb8290
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.10+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.18+c6af733c90
			fabric-block-api-v1: Fabric Block API (v1) 1.0.0+12bfe4ea90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.21+c6af733c90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.2+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.12+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.8+93d8cb8290
			fabric-commands-v0: Fabric Commands (v0) 0.2.29+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.35+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.1+6064db2a90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.2+93d8cb8290
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.6+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.2.3+b598f4ac90
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.32+0dd10df690
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.0+a1d87cb890
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.29+c6af733c90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.29+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.22+c6af733c90
			fabric-item-api-v1: Fabric Item API (v1) 1.5.8+93d8cb8290
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.33+70efcffe90
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.22+aaaf9d3390
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.20+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.0+33ffe9ec90
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.7+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.10+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.4+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.18+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.18+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.5+c6af733c90
			fabric-networking-v0: Fabric Networking (v0) 0.3.22+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.1.0+06e0e25190
			fabric-particles-v1: Fabric Particles (v1) 1.0.11+79adfe0a90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.26+c6af733c90
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.13+12bfe4ea90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.15+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.21+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.16+12bfe4ea90
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.8+c6af733c90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.23+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.0+73145abb90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.0+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.7.3+3e1ba48e90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.27+93d8cb8290
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.1+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.0+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.21+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.1+93d8cb8290
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.1+42d99c3290
		fabricloader: Fabric Loader 0.14.10
		fwaystones: Fabric Waystones 3.0.5+mc1.19.2
		image2map: Image2Map 0.3.0
			map-canvas-api: Map Canvas API 0.2.0+1.19.1
			sgui: sgui 1.1.5+1.19.1
		java: OpenJDK 64-Bit Server VM 17
		latoorigins: Lato Origins 1.19.2-1.1.0
		minecraft: Minecraft 1.19.2
		moborigins: Mob Origins 1.9.0
		moonlight: Moonlight 1.19.2-2.1.10
		origins: Origins 1.7.1
			apoli: Apoli 2.6.1
				additionalentityattributes: Additionalentityattributes 1.0.0+1.19
				calio: Calio 1.7.0
				cardinal-components-base: Cardinal Components API (base) 5.0.1
				cardinal-components-entity: Cardinal Components API (entities) 5.0.1
				cloth-config: Cloth Config v8 8.0.75
					cloth-basic-math: cloth-basic-math 0.6.1
				playerabilitylib: Pal 1.6.0
			reach-entity-attributes: Reach Entity Attributes 2.3.0
		originsumbrellas: Origins: Umbrellas 1.5.4
		owo: oÏ‰o 0.9.0+1.19
			blue_endless_jankson: jankson 1.2.1
		pehkui: Pehkui 3.6.0+1.14.4-1.19.3
			kanos_config: Kanos Config 0.3.1+1.14.4-1.19
		sit: Sit 1.19-20
		supplementaries: Supplementaries 1.19.2-2.2.21
		toomanyorigins: TooManyOrigins 0.6.7+1.19-fabric
			apugli: Apugli 1.10.0+1.19-fabric
				com_github_llamalad7_mixinextras: MixinExtras 0.1.0
				midnightlib: MidnightLib 0.5.2
		whisperwoods: Whisperwoods 1.19-2.1.1
		worldedit: WorldEdit 7.2.12+6240-87f4ae1
		xercamusic: Music Maker Mod fabric-1.19.2-1.0.0
	Launched Version: fabric-loader-0.14.10-1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

**Comment by modmuss50** [comment]
Try deleting the `.minecraft/config/image2map.json` file. It appears to have invalid json.

**Reply by n0tcharli3** [reply]
did that, still nothing :/

**Comment by warjort** [comment]
From where it crashes, it looks like an issue with the croptopia (biome?) configuration file?
Since it fails at the first character in the file, it is probably corrupted.
You should contact the mod author to ask how to fix it.

Or you can try deleting it. Usually if you delete a configuration file, the mod will recreate it with default values.
The error message doesn't say what the file is called though, so you will probably still have to talk to the mod author to find out the name of the file.
